Userland/Libraries/LibCore/StandardPaths.cpp

```diff
@@ -117,6 +136,10 @@ ByteString StandardPaths::videos_directory()
 
 ByteString StandardPaths::config_directory()
 {
+#ifdef AK_OS_WINDOWS
+    dbgln("Core::StandardPaths::config_directory() is not implemented");
+    VERIFY_NOT_REACHED();
+#endif
```

Comment:
with the same into as above:

not sure what it used for, but if it is just to keep ladybird configuration, `%LocalAppData%/Ladybird` could be a decent storage path for that.